apps/trading-e2e/src/integration/deposits.feature

```diff
@@ -0,0 +1,112 @@
+Feature: Deposits to vega wallet
+
+  Background:
+    Given I navigate to deposits page
+
+  Scenario: Eth wallet connect message displayed if no ethereum wallet connected
+    Then I can see the eth not connected message "Connect your Ethereum wallet"
+    And the connect button is displayed
+
+  @todo
+  Scenario: Cannot deposit if approved amount is 0 (approval amount is 0)
+    And I connect my ethereum wallet
+    And I connect my vega wallet
+    When I set "0" tokens to be approved
+    And I approve the asset tokens
+    And I can see the deposit form is displayed
+    And I select "" asset from the dropdown list
+    When I enter the following details
+      | Field         | Value    |
+      | To (Vega key) | xxxxxxxx |
```

Comment:
question (non-blocking): @jtsang586 Can we use the env vars you've setup for these vega keys?

Reply:
I don't think that's possible from the feature file. 
Here I would create a separate step for different public keys e.g. 
1. using key stored in Cypress env 
2. invalid key etc.

Reply:
we can't call any logic in the feature files . we would need to do it in the step definition

Reply:
Can we use a magic string then?
.feature
```
| Field         | Value    |
| To (Vega key) | current |
```

.page.js

```
if (ket === 'current') key= cy.getEnv('VegaKey')
```

Not sure of the exact syntax but I would like anyone to be able to run these tests without needing a specific wallet, which will make capsule integration easier